website/source/api/index.html.md

```diff
@@ -196,3 +196,14 @@ GET /v1/jobs
 
 Even though these share a path, the `PUT` operation creates a new job whereas
 the `GET` operation reads all jobs.
+
+## HTTP Response Codes
+
+Individual API's will contain further documentation in the case that more
+specific response codes are returned but all clients should handle the following:
+
+* 200 and 204 as success codes.
+* 4xx codes indicate that if a parameter was modified in the request, it could
+  potentially succeed.
```

Comment:
Took me a second to parse this. Should be 400 instead of 4xx right? Seems funny to say you can fix a 404 by changing a parameter even if that's kinda sorta true.

Also maybe mention it indicates a validation failure:

```
400 indicates a validation failure and if a parameter is modified in the request, it could potentially succeed.
```

I guess you could argue not all 400s are technically "validation", but I still think it's clearer this way.

Reply:
Hmm while technically correct, I see your point